common/decorators/__init_.py

```diff
@@ -0,0 +1,18 @@
+
+import numpy as np
+
+def scalar_input(func):
```

Comment:
Are we using this at all?

Reply:
Not yet because Astropy structures makes it a little harder. I left it there as a remainder that could be a good idea to implement.

Reply:
👍 I thought that might be the case because I think a while ago, I tried to do something like this and it didn't go as smoothly as I anticipated it to go. Okay, let's leave it in and see what happens, and if we don't use it, we'll take it out later.